.gitignore

```diff
@@ -1,4 +1,6 @@
 /lib/
 /.env
 /testbox/
-/.vscode/
\ No newline at end of file
+/.vscode/
+
+tests/specs/LDTestLive.cfc
```

Comment:
Note - I had added this file because I wanted a way to test against our live LD instance with a subset of customized tests, but didn't want to commit these to the code base (as they would be customized for our flag collection).  Just added it here to make sure the file wasn't committed by accident.  Feel free to leave / remove as appropriate